trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorContext.java

```diff
@@ -42,7 +42,7 @@ public CatalogConnectorContext(
     this.internalConnector = internalConnector;
     this.adapter = adapter;
 
-    this.connector = new GravitinoConnector(catalogName, this);
+    this.connector = new GravitinoConnector(this.catalogName, this);
```

Comment:
@justinmclean  I think `catalogName` is necessary for `GravitinoConnector`, which can not remove, so I change this by using `'this'`

Reply:
The is that while this.catalogName is set, it is never used and as far as I can see there no need for the field at all.

Reply:
I would remove the `catalogName` field as it is not used anywhere else that I can see